keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -263,10 +264,51 @@ metadata:
             values:
             - database
         topologyKey: topology.kubernetes.io/zone
-        matchlabelKeys: # ADDED
-        - pod-template-hash
+        matchLabelSelectors: # ADDED
+        - matchLabelKeys: pod-template-hash
+          operator: In
 ```
 
+#### Story 2
+
+When users want to achieve exclusive 1:1 job to domain placement in [kubernetes-sigs/jobset](https://github.com/kubernetes-sigs/jobset).
```

Comment:
Great. their common key with the orchestrator are different value about Mpod&Npod.Can.we avoid this problem by specifying pod for foo？

Reply:
OK, I'll try to have another example here to explain the same usecase.

Reply:
As you suggested, I changed this example to be more general, not using jobset. PTAL.

I didn't follow your example because I feel like users can define LabelSelector of `foo=abc` / `foo=xyz` directly in Pod affinity / anti affinity. I think the point of the usecase of matchLabelKey is that "users cannot know the label value because, for example, the label value is applied from another system automatically (like jobset controller for `job-name`, the deployment controller for `pod-template-hash`, generated in the manifest manager, etc)".